src/views/HomePageView/ExclusiveActivity/ExclusiveActivity.styles.js

```diff
@@ -0,0 +1,50 @@
+// vendors
+import styled, { css } from 'styled-components';
+
+// utils
+import breakpointsRange from '../../../utils/breakpointsRange';
+
+// styles
+import breakpoints from '../../../styles/breakpoints';
+import colors from '../../../styles/colors';
+import { fontWeights } from '../../../styles/typography';
+
+export const ActivityWrapper = styled.div`
```

Comment:
Comme pour les autres sections de la page d'accueil
```suggestion
export const ActivityWrapper = styled.section`
```